fabric-content-registries-v0/src/main/java/net/fabricmc/fabric/api/registry/FabricBrewingRecipeRegistry.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.registry;
+
+import java.util.Objects;
+
+import net.minecraft.item.Item;
+import net.minecraft.item.PotionItem;
+import net.minecraft.potion.Potion;
+import net.minecraft.recipe.BrewingRecipeRegistry;
+import net.minecraft.recipe.Ingredient;
+
+import net.fabricmc.fabric.mixin.content.registry.BrewingRecipeRegistryAccessor;
+
+/**
+ * Counterpart of {@link BrewingRecipeRegistry} with methods that allow adding recipes which use Ingredients instead of Items.
+ */
+public final class FabricBrewingRecipeRegistry {
+	private FabricBrewingRecipeRegistry() {
+	}
+
+	/**
+	 * Register a recipe for brewing one potion type into another (e.g. regular to splash).
+	 * Only one registration is necessary for all potions of the input type to be brewable into the output type.
```

Comment:
I feel like there should be documentation on how to register the new potion item (call registerPotionType).
```suggestion
	 * Only one registration is necessary for all potions of the input type to be brewable into the output type.
	 * A new potion type should be registered with {@link BrewingRecipeRegistry#registerPotionType(Item)} before calling this.
```